src/components/header-menu/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import HeaderMenuItem from '../header-menu-item';
+import { HeaderMenuProps } from './types';
+import LogoutButton from '../logout-button';
+import './header-menu.css';
+
+const HeaderMenu = ({ headerMenu }: HeaderMenuProps) => {
+    return (
+        <div className="header-menu">
```

Comment:
И потом использовать как

```suggestion
        <div className={css['header-menu']}>
```

Редактор в таком случае подскажет имена стилей и подсветит, если есть опечатка в имени класса. Ну и для удобства можно именовать css-классы в `camelCase`

```javascript
        <div className={css.headerMenu}>
```